File: Miscelaneous/livro.py
class Livro:
    def __init__(self, titulo, autor):
        self.__titulo = titulo
        self.__autor = autor
        self.__disponivel = True

    @property
    def titulo(self):
        return self.__titulo

    @property
    def autor(self):
        return self.__autor

    def emprestar(self):
        if not self.__disponivel:
            return False
        self.__disponivel = False
        return True

    def devolver(self):
        self.__disponivel = True

File: Miscelaneous/Livro.java
public class Livro {
    private final String titulo;
    private final String autor;
    private boolean disponivel = true;

    public Livro(String titulo, String autor) {
        this.titulo = titulo;
        this.autor = autor;
    }

    public String getTitulo() { return titulo; }
    public String getAutor() { return autor; }

    public boolean emprestar() {
        if (!disponivel) return false;
        disponivel = false;
        return true;
    }

    public void devolver() { disponivel = true; }
}